### DISCUSSION TITLE
如果两个独立的FE, 添加同一个BE, 两个FE是否都可以正常提供服务呢?

### DISCUSSION THREAD

**Discussion by isCoding** [apache/doris#20755] [RFC]
有一个独立的FE节点, 机房要不可用一段时间, 想要换一个FE节点, 不知道直接挂载现存的3个be, 会不会导致现有集群异常.
版本 1.2.1

**Comment by yagagagaga** [comment]
你应该在集群里添加一个FE节点，然后把老的FE节点下线掉，新的FE节点自动成为master，Doris正常工作。

**Reply by isCoding** [reply]
谢谢, 我按照这个操作了
